src/xmidtsend_rbus.c

```diff
@@ -45,6 +45,10 @@ const char * contentTypeList[]={
 "application/binary"
 };
 
+void printSendMsgData(char* status, char* dest, int qos, char* transaction_uuid) {
+	ParodusInfo("Status of event: %s, destination: %s, qos: %d, transation_uuid: %s\n", (status!=NULL)?status:"NULL", (dest!=NULL)?dest:"NULL", qos, (transaction_uuid!=NULL)?transaction_uuid:"NULL");
```

Comment:
Currently log added is -
`220615-14:06:56.031367 [mod=PARODUS, lvl=INFO] [tid=10982] PARODUS: Status of event: sendUpstreamMsgToServer after cloud reconnect, destination: event:/profile-notify/MyProfile1, qos: 67, transation_uuid: d00d6632-bc24-4eb4-bba5-c1704c6317bd`

can we shorten this log a bit:
`PARODUS: status: send to server after cloud reconnect, qos: 67, dest: event:/profile-notify/MyProfile1, transaction_uuid: d00d6632-bc24-4eb4-bba5-c1704c6317bd`

Reply:
changes are done